osbs/cli/capture.py

```diff
@@ -20,18 +20,26 @@ class IterLinesSaver(object):
     Wrap HttpStream.iter_lines() and save responses.
     """
 
-    def __init__(self, path, fn):
+    def __init__(self, encoding, path, fn):
+        self.encoding = encoding or 'utf-8'
         self.path = path
         self.fn = fn
         self.line = 0
 
     def iter_lines(self):
+        encoding = self.encoding
         for line in self.fn():
             path = "{f}-{n:0>3}.json".format(f=self.path, n=self.line)
             logger.debug("capturing to %s", path)
+            try:
+                text = line.decode(encoding, errors='replace')
```

Comment:
I'm not sure if we should `replace` by default - all our parts should be able to handle Unicode in logs. Maybe we could try decoding it as is and falling back to `errors='replace'` in case exception occurs?